lib/vsc/administration/user.py

```diff
@@ -209,10 +209,11 @@ def user_proposition(quota, storage_type):
 
         # Non-UGent users who have quota in Gent, e.g., in a VO, should not have these set
         if self.person.institute['name'] == self.host_institute:
-            self._cache['quota']['home'] = [q.hard for q in institute_quota if user_proposition(q, HOME_KEY)][0]
-            self._cache['quota']['data'] = [q.hard for q in institute_quota
+            self._cache['quota']['home'] = next(iter([q.hard for q in institute_quota
```

Comment:
so what is this `next(iter(` construct about?

Reply:
it's for: get first item or None

Reply:
the orginal error failed on the `[0]` at the end of the line.
With this construct of `next(iter(` we can get the first item of the list if it is non-empty or `None` otherwise.

Reply:
I can add a comment explaining?

Reply:
yes, it's just looks odd. i thought that `pop` had a default value option.

Reply:
pop doesn't have a default value thingie.